gorm/fields.go

```diff
@@ -75,36 +77,55 @@ fields:
 	return toPreload, nil
 }
 
-func handlePreloads(f *query.Field, objType reflect.Type) ([]string, error) {
-	sf, ok := objType.FieldByName(util.Camel(f.GetName()))
+func handlePreloads(f *query.Field, objType reflect.Type, ignoreCase ...bool) ([]string, error) {
+	queryFieldName := f.GetName()
+	fmt.Printf("Query name = %v\n", queryFieldName)
+
+	var sf reflect.StructField
+	var ok bool
+	if len(ignoreCase) > 0 && ignoreCase[0] {
```

Comment:
maybe I'm missing something but why is `ignoreCase` a `...bool` instead of just a plain old `bool`?

Reply:
This is an "optional" parameter for backward compatibility.

Reply:
ok that makes sense. if that's the case then make sure that there is a unti test for `ignoreCase` being `false`, and one for there being no `ignoreCase` value at all

Reply:
Yes, there are. In test file, the function has been called as followed
	FieldSelectionStringToGorm(context.Background(), test.fs, &Model{}) // no "ignoreCase" = false
	FieldSelectionStringToGorm(context.Background(), test.fs, &Model{}, true)

Reply:
it might be nice to have at least one `FieldSelectionStringToGorm(context.Background(), test.fs, &Model{}, false)` just for completion's sake